src/Categories/Bicategory/Construction/LaxSlice.agda

```diff
@@ -0,0 +1,337 @@
+{-# OPTIONS --without-K --safe #-}
+
+-- Mentioned in passing https://ncatlab.org/nlab/show/slice+2-category
+
+open import Categories.Bicategory
+
+module Categories.Bicategory.Construction.LaxSlice
+       {o ℓ e t}
+       (𝒞 : Bicategory o ℓ e t)
+       where
+
+open import Categories.Enriched.Category
```

Comment:
The more modern parts of agda-categories tend to be quite specific on all imports (unless there's tons and tons from one file).